src/lib.rs

```diff
@@ -1,4 +1,4 @@
-mod core;
+pub mod core;
```

Comment:
Do not expose core publicly. Control the things you wish to expose publicly down below.